c++/src/kj/iovmax.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2013-2014 Sandstorm Development Group, Inc. and contributors
```

Comment:
Actually, `miniposix.h` isn't intended to be a public API itself, and arguably it should not be installed with the rest of the headers.

No need for a Win32 iovMax() since Win32 doesn't have iov in the first place, so no one would ever call it.